assets/scripts/dialogs/AnalyticsDialog.jsx

```diff
@@ -82,11 +82,10 @@ function AnalyticsDialog (props) {
     }}
   />)
 
-  const getRolledCapacity = item => Number.isInteger(item.capacity.potential) ? item.capacity.potential : 0
-  const hasRolledCapacity = item => Number.isInteger(item.capacity.potential)
+  const displayCapacity = item => item.capacity && item.capacity.display && item.capacity.average > 0
```

Comment:
I'm excluding segments that would normally have capacity but that don't due to errors here. Let me know if that's not the right behavior?

Reply:
Is this also excluding segments that are too wide, but otherwise fit in the street?

Segments that are too wide don't drop their capacities to zero. So they should still be included.

Reply:
Nope, those segments do not have the requisite error type to set their capacity capacity to zero. See here: https://github.com/streetmix/streetmix/blob/77a557155f3730aeb7e913e3eaa4666cb2ec154a/assets/scripts/util/street_analytics.js#L74